webhooks.py

```diff
@@ -193,4 +201,4 @@ def index():
 
 
 if __name__ == '__main__':
-    application.run(debug=True, host='0.0.0.0')
+    application.run(debug=False, host='0.0.0.0', port=8080)
```

Comment:
Standalone deamon on the metal. I wrote upstart config for that, and forward nginx upstream to gunicorn.